gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,174 @@
+      </NxP>
+      <NxWarningAlert>
+        Note: In order to render toasts in the correct position, all toasts must be wrapped within the parent React
+        component <NxCode>NxToastContainer</NxCode>, which must be a direct child of <NxCode>NxPageMain</NxCode>.
+      </NxWarningAlert>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>NxToastContainer</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Resource</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>children</NxTable.Cell>
+              <NxTable.Cell>React Node</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                All <NxCode>NxToast</NxCode>s rendered within <NxCode>NxToastContainer</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>NxToast</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Resource</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>onClose</NxTable.Cell>
+              <NxTable.Cell>Function</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                A function that dismisses the toast when called. When the close button is clicked, this callback
+                will be fired.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>toastId</NxTable.Cell>
+              <NxTable.Cell>number</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                A unique numerical identifier for each toast that is used in the onClose callback to identify and
+                remove the correct toast from the page.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>children</NxTable.Cell>
+              <NxTable.Cell>React Element</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                A single <NxCode>NxAlert</NxCode>, or any of its four variations. Note that each
+                {' '}<NxCode>NxToast</NxCode> will only accept one <NxCode>NxAlert</NxCode>.
```

Comment:
I think there are two additional things that should be noted here:
1. it doesn't have to be an `NxToast` so much as something that ultimately renders an `NxToast` and which accepts an `onClose` handler.  For instance if a page had a `MySpecificToast` component which rendered an `NxErrorAlert` with particular text, that could be used here as well.
2. The text of the toast should only be one line long. (FWIW I tried it with longer text and it does largely work, the only issue being that the glide up and down parts of the animation are no longer smooth as it basically jumps between being the height needed for a single-line toast and the actual height of the toast.

Reply:
> I think there are two additional things that should be noted here:
> 
> 1. it doesn't have to be an `NxToast` so much as something that ultimately renders an `NxToast` and which accepts an `onClose` handler.  For instance if a page had a `MySpecificToast` component which rendered an `NxErrorAlert` with particular text, that could be used here as well.
> 2. The text of the toast should only be one line long. (FWIW I tried it with longer text and it does largely work, the only issue being that the glide up and down parts of the animation are no longer smooth as it basically jumps between being the height needed for a single-line toast and the actual height of the toast)

I added into the docs here: [bba523c](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/bba523c7b55c4033284813bb082371eb38ba9d08). I'm not sure if my explanation for the first comment is on point, so feel free to have me re-write if necessary.

Reply:
> How about
> 
> > A single `ReactElement` which accepts and `onClose` prop and which ultimately renders an `NxAlert` .....

Well said! Updated here: [d52e888](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/d52e88814199936fde741261e7ffb7673a1ac3a7)